pkgs/tools/admin/salt/default.nix

```diff
@@ -1,21 +1,43 @@
-{
-  stdenv, pythonPackages, openssl,
-
-  # Many Salt modules require various Python modules to be installed,
+{ lib
+, python3
+, openssl
+, # Many Salt modules require various Python modules to be installed,
   # passing them in this array enables Salt to find them.
   extraInputs ? []
 }:
+let
+
+  defaultOverrides = [
+    # can be unpinned once https://github.com/saltstack/salt/issues/56007 is resolved
+    (mkOverride "msgpack" "0.6.2"
+      "0c0q3vx0x137567msgs5dnizghnr059qi5kfqigxbz26jf2jyg7a")
+  ];
+
+  mkOverride = attrname: version: sha256:
```

Comment:
Any reason not to inline this (and also to inline `defaultOverrides` into `py = ...` below?

Reply:
No specific reason, I just copied the code from somewhere else, though I agree that it’s overkill for a single package override.